src/test/features/dashboard/helpers/claimStatusFlow.ts

```diff
@@ -0,0 +1,135 @@
+import { expect } from 'chai'
+import { Claim } from 'claims/models/claim'
+import { sampleClaimObj } from '../../../http-mocks/claim-store'
+import { ClaimStatusNode, ClaimStatusFlow } from 'dashboard/helpers/claimStatusFlow'
+
+describe('The dashboard status rule engine', () => {
+  describe('given the claim flow', () => {
+    it('should extract the correct dashboard for claim issued', () => {
+      const claim: Claim = new Claim().deserialize(sampleClaimObj)
+      expect(ClaimStatusFlow.dashboardFor(claim)).to.equal('claim_issued')
+    })
+  })
+
+  describe('given a generic flow', () => {
+    it('should return the dashboard of the only valid state', () => {
+      const flow: ClaimStatusNode = {
+        description: 'this is always true',
+        isValidFor: () => true,
+        dashboard: 'first',
+        next: []
+      }
+      const claim: Claim = new Claim().deserialize(sampleClaimObj)
+      expect(ClaimStatusFlow.decide(flow, claim)).to.equal(`first`)
+    })
+  })
```

Comment:
Please add line spacings between tests just for readability (We follow this in other tests). Everything else looks good..

Reply:
👍